### ISSUE TITLE
update to pathology source data

### ISSUE THREAD

**Issue by annagree** [Sage-Bionetworks/magora#27]
We need to update the source data for the pathology explorer. It will require pulling data from Synapse, similarly to what is currently being done for gene expression data.

**Comment by annagree** [comment]
Note we are currently waiting on details from our collaborator, so this is on hold for now.

**Comment by annagree** [comment]
Mapping of new phenotypes from new source files is shown in this table:
https://drive.google.com/file/d/1L56zZW4Qa37gqc3zXw_Xm9QLR9SQ6wLA/view?usp=sharing

The associated metadata is found in these files:
syn18876530
syn18880070

**Comment by karawoo** [comment]
@sharlagelfand as a first step, could you please compare the individual IDs in the data to the metadata and make note of any that are missing from the metadata?

**Comment by annagree** [comment]
Responding to a couple of issues in the Slack channel:

_For https://www.synapse.org/#!Synapse:syn22049817, the google sheet indicates to use `# objects/sqmm` for Plaque density / LAMP1 + Cell Density. This column doesn’t exist - we can derive it using using two columns: `# objects` and `Area (square mm )`, but `# objects` are all NA. In the same file, `Mean object area (square mm)` (which is used for Plaque size / LAMP1 + Cell Size) are also all NA._

For plaque density and plaque size, we should actually use syn22049825, with this condition: Stain= ThioS

**Comment by annagree** [comment]
This also from the Slack channel:
_For https://www.synapse.org/#!Synapse:syn22049819, the column `mean object area (sqmm)` (used for NeuN+ Cell Size) does not exist._

In this file, the column is called "area" instead of "mean object area".

**Comment by annagree** [comment]
I've also updated the mapping information in this file:
https://docs.google.com/spreadsheets/d/1mpk1JFp9oF5du1qqfKK5xVYUftWSRHbiycJx7m2uuAY/edit#gid=162516105

**Comment by sharlagelfand** [comment]
Thanks @annagree! I've looked at these changes, a couple more questions:

> For plaque density and plaque size, we should actually use syn22049825, with this condition: Stain= ThioS

Thanks, this looks good for Plaque Density and Size now!

Is there a different file we should use for LAMP1+ Cell Density and Size as well? Since they are using the data source that Plaque density and size previously used, the issue of the missing columns/NA data is still present for those.

With this change to syn22049825 for Plaque density and size, now Microglia Density and Size are using the same data and fields as Plaque. Should there be a Stain filter for Microglia (Plaque has Stain = "ThioS", but Microglia uses all of the data)?

> In this file, the column is called "area" instead of "mean object area".

Thank you - looks good now!

I missed one issue before that I'll flag now - all values for S100b+ Cell Size are `NA`. This uses the same data set / field as GFAP+ Cell Size, but with a different filter for Stain. There is data for GFAP+ Cell size, just all `NA`s for S100b+ - may just be how the data actually is, but wanted to flag that there are no values there.

Thank you!

**Comment by annagree** [comment]
@sharlagelfand Hi Sharla. I got some additional clarification from our data contributor. One thing I told you above is incorrect, and there are some additional clarifications. 

The issues stem from the fact that these variables are not always measurable in all cases. I originally thought we would have Cell Size and Cell Density for all of the different stains. However, there are several stains where that isn't true. In those cases, we either won't report one of these variables (Stain = NeuN; Stain = S100B). Or we will report something else instead (Stain = LAMP1). 

I've updated this in this spreadsheet, by striking through some variables we won't be using and adding one new one-- let me know if you have any additional questions after you get a chance to check it out:
https://docs.google.com/spreadsheets/d/1mpk1JFp9oF5du1qqfKK5xVYUftWSRHbiycJx7m2uuAY/edit#gid=162516105

**Comment by sharlagelfand** [comment]
Thanks @annagree! All looks good with those data sources now.

Here's the list of `individualID`s that are missing from the [individual metadata](https://www.synapse.org/#!Synapse:syn18880070), so it doesn't just live and get lost in Slack :)

```
4255
4503
4244
4209
4208
4204
4281
4199
4266
4265
4264
4242
```

**Comment by annagree** [comment]
@sharlagelfand The team has updated the individual metadata file so if you download the latest version, it should now include those mice.

**Comment by sharlagelfand** [comment]
Thanks @annagree! Have updated to the latest version, looks great.
